src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -81,6 +81,10 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
         context_path = None
         commit_trigger_enabled = False
         pull_request_trigger_enabled = False
+    
+    if context_path.lower().startswith("oci://"):
```

Comment:
@akashsinghal this will throw a null exception if the above case is met and we set `context_path = None`. Please add a null check here before calling `.lower()`